awscli/customizations/ec2/paginate.py

```diff
@@ -0,0 +1,92 @@
+# Copyright 2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+
+class EC2PageSizeInjector(object):
+
+    DEFAULT_TARGET_OPERATIONS = {
+        "describe-volumes": [],
+        "describe-snapshots": ['owner-ids', 'restorable_by_user_ids']
```

Comment:
Why does ``owner-ids`` have a hyphen and ``restorable_by_user_ids`` uses underscores?